docs/pages/reference/troubleshooting/installation.md

```diff
@@ -20,6 +20,25 @@ Anytime a `mesheryctl system` command is executed, a series of preflight checks
 
     Situation: Unable to start Meshery Server with `make run-local` due to error of `key/value size is invalid`
 
+## Setting up Meshery using Kind or Minikube
+
+The difficulty with Minikube and Kind clusters is that they typically don't have a loadbalancer, whereas Meshery gives its users complete control over how networking is configured. As a result, networking for these deployments is more complex. Few of the alternatives are:
```

Comment:
If "loadbalancer" in this sense refers to the Kubernetes construct, please capitalize the "L" and "B".